types/variables.go

```diff
@@ -5,4 +5,4 @@ package types
 
 // VariablesCount contains the number of variables handled by the variables package
 // It is used to create arrays of the correct size
-const VariablesCount = 91
+const VariablesCount = 92
```

Comment:
I think this file should be generated, what do you think? Wanna give a try @M4tteoP ?